app/src/main/res/values/strings.xml

```diff
@@ -289,4 +289,20 @@
   <string name="wikidata_edit_failure">Failed to update corresponding Wikidata entity!</string>
   <string name="menu_set_wallpaper">Set wallpaper</string>
   <string name="wallpaper_set_successfully">Wallpaper set successfully!</string>
+  <string name="quiz">Quiz</string>
+  <string name="quiz_question_string">Is this picture OK to upload?</string>
+  <string name="question">Question</string>
+  <string name="result">Result</string>
+  <string name="quiz_back_button">End quiz ? </string>
```

Comment:
Ah, I just realized this string is displayed when the user tries to prematurely end the quiz. In that case I think more explanation is in order, perhaps "If you carry on uploading images that require deletion, your account will likely be banned. Are you sure you want to end the quiz?".